frontends/election-manager/src/components/smartcard_modal/status_message.tsx

```diff
@@ -126,28 +126,25 @@ export function InProgressStatusMessage({
   actionStatus,
 }: InProgressStatusMessageProps): JSX.Element | null {
   const { action } = actionStatus;
-  const actionRoleReadableString = userRoleToReadableString(actionStatus.role);
 
   let text: string;
   switch (action) {
     case 'Program': {
-      text = `Creating ${actionRoleReadableString} Card`;
+      text = 'Programming card';
       break;
     }
     case 'PinReset': {
-      text = `Resetting ${actionRoleReadableString} Card PIN`;
+      text = 'Resetting card PIN';
       break;
     }
     case 'Unprogram': {
-      text = `Unprogramming ${actionRoleReadableString} Card`;
+      text = 'Unprogramming card';
       break;
     }
     /* istanbul ignore next: Compile-time check for completeness */
     default: {
       throwIllegalValue(action);
     }
   }
-  return (
-    <Modal centerContent content={<Loading as="strong">{text}</Loading>} />
-  );
+  return <Modal centerContent content={<Loading>{text}</Loading>} />;
```

Comment:
Now that we don't have user roles in our in-progress strings, we can make them larger! (By default, `Loading` uses an `h1`, so larger than the current `strong`)